### ISSUE TITLE
PDBList.py update_pdb does not consider multiple file formats (in some parts...)

### ISSUE THREAD

**Issue by JoshuaMeyers** [biopython/biopython#1691]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```python
import sys; print(sys.version)
import platform; print(platform.python_implementation()); print(platform.platform())
import Bio; print(Bio.__version__)
```
```3.6.1 |Anaconda custom (64-bit)| (default, May 11 2017, 13:04:09) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)]
CPython
Darwin-16.7.0-x86_64-i386-64bit
1.71
```

The PDBList.py method `update_pdb` is extremely useful, but just requires some minor updating!

### Expected behaviour

1. It should move obsoleted pdb files to the specified path
2. Currently the function does not return anything

### Actual behaviour

1. The part of the function which moves obsolete pdb entries has not kept up with the change to accept multiple `file_format`'s. Therefore it checks only for `pdb{pdb_code}.ent` and returns `Obsolete file [...]PDB/tt/pdb5tti.ent is missing` (even if `.ent` isn't the specified `file_format`)
2. It would also be useful to return the updates that have been applied (to provide a diff)

### Steps to reproduce

`pdbl = PDBList(server='ftp://ftp.wwpdb.org',
               pdb='/PDB',
               obsolete_pdb='/PDB/obsolete',
               verbose=True)
pdbl.update_pdb()`

**Comment by peterjc** [comment]
@JoshuaMeyers can you see how to fix this, and submit a pull request please?

If not, @dadoskawina would you like to work on this as a followup to your contributions in #943?

Thank you both.

**Comment by poleshe** [comment]
Hi!

If this hasn't been fixed yet, I can tackle it. We'll see how it goes.

Cheers,

**Comment by peterjc** [comment]
Yes please @poleshe - please start be re-testing to confirm it the problem is still there.

**Comment by poleshe** [comment]
Greetings, 
I can confirm this still happens. 
update_pdb is still only searching for ".ent" files, even after passing a "file_format" attribute.

`pdbl.update_pdb(file_format="xml")`

![pdb](https://user-images.githubusercontent.com/47125355/150613609-4e45a384-f395-42fc-b21b-0575a62f72ac.PNG)

![missing](https://user-images.githubusercontent.com/47125355/150613610-d9a4ab54-cd0a-48ef-97e0-9b46d8ffbf2f.PNG)

Will develop a fix for this. 

Have a nice weekend!

**Comment by poleshe** [comment]
Thanks @pycreatine :)
